docscan/session/create/check/liveness.go

```diff
@@ -36,34 +49,33 @@ func NewRequestedLivenessCheckBuilder() *RequestedLivenessCheckBuilder {
 type RequestedLivenessCheckBuilder struct {
 	livenessType string
 	maxRetries   *int
-	//TODO: do we want this with pointer so we can set to nil?
 }
 
 // ForZoomLiveness sets the liveness type to "ZOOM"
-func (builder *RequestedLivenessCheckBuilder) ForZoomLiveness() *RequestedLivenessCheckBuilder {
-	return builder.ForLivenessType(constants.Zoom)
+func (b *RequestedLivenessCheckBuilder) ForZoomLiveness() *RequestedLivenessCheckBuilder {
+	return b.ForLivenessType(constants.Zoom)
```

Comment:
I noticed you declared a separate `constants` package. In Go, normally constants/enums should be declared close to where they are relevant.

It's also worth considering using type aliases:
```
type LivenessType string
const (
   Zoom LivenessType = "ZOOM"
)
```
And then make the value in the struct be of that type to prevent some rogue person accidently assigning "ID_DOCUMENT" to liveness.

Reply:
I've made this change in 1574508, I've left them unexported if they don't need to be, and we can always make them exported at some point before releasing if need be.